src/bloqade/task/braket_simulator.py

```diff
@@ -19,17 +24,48 @@
 @dataclass
 @Serializer.register
 class BraketEmulatorTask(LocalTask):
+    """
+    Represents a quantum task that runs on the AWS Braket LocalSimulator.
+
+    Attributes:
+        task_ir (BraketTaskSpecification): The task specification for the Braket simulator.
+        metadata (Dict[str, ParamType]): Metadata related to the task.
+        task_result_ir (Optional[QuEraTaskResults]): The results of the task, if available.
+    """
+
     task_ir: BraketTaskSpecification
     metadata: Dict[str, ParamType]
     task_result_ir: Optional[QuEraTaskResults] = None
 
     def _geometry(self) -> Geometry:
+        """
+        Constructs and returns the Geometry of the task based on the AHS register setup.
+
+        Returns:
+            Geometry: The geometric setup of the task.
+        """
         return Geometry(
             sites=self.task_ir.program.setup.ahs_register.sites,
             filling=self.task_ir.program.setup.ahs_register.filling,
         )
 
     def run(self, **kwargs) -> "BraketEmulatorTask":
+        """
+        Runs the task on the Braket LocalSimulator.
+
+        Args:
+            **kwargs: Additional arguments for the simulator run method.
+
+        Returns:
+            BraketEmulatorTask: The current instance with the task results updated.
+
+        Example:
```

Comment:
same as my previous comment, this needs to use a markdown code block

Reply:
also can you actually run these code and copy the important printings back, e.g the printing of task and result

Reply:
Just for clarity you just need to do:

```python
"""

Example:
    ```python
    >>> <example code>
    >>> ...
    ```

"""
```